ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -54,7 +55,27 @@
     "ipahealthcheck.ipa.roles",
     "ipahealthcheck.ipa.topology",
     "ipahealthcheck.ipa.trust",
+    "ipahealthcheck.ipa.meta",
+    "ipahealthcheck.meta.core",
     "ipahealthcheck.meta.services",
+    "ipahealthcheck.system.filesystemspace",
+]
+
+sources_0_4 = [
+    "ipahealthcheck.dogtag.ca",
+    "ipahealthcheck.ds.replication",
+    "ipahealthcheck.dogtag.ca",
```

Comment:
"ipahealthcheck.dogtag.ca" appears twice in source and sources_0_4, can you simplify and remove the duplicate?